realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/AuthTests.java

```diff
@@ -447,6 +446,21 @@ public void singleUserCanBeLoggedInAndOutRepeatedly() {
         }
     }
 
+    // verify that multiple users can be logged in at the same time
```

Comment:
Seems the Cartasite issue was captured by another unit test by @beeender. Perhaps we should just close this PR?